tests/h/auth/util_test.py

```diff
@@ -183,6 +185,78 @@ def test_it_returns_text_type(self, pyramid_request):
         assert type(util.authority(pyramid_request)) == text_type
 
 
+class TestValidateAuthClientAuthority(object):
+
+    def test_raises_when_authority_doesnt_match(self, pyramid_request, auth_client):
```

Comment:
I think we should add an additional test here—the existing test tests for a failure path, but there's no happy path test. Could you add a test that it allows when the authority _does_ match? Thanks!